.github/ISSUE_TEMPLATE/-support-request.md

```diff
@@ -14,6 +14,6 @@ GitHub is not the right place for support requests.
 
 If you're looking for help, check our [docs](https://github.com/kubernetes-sigs/external-dns/tree/HEAD/docs).
 
-You can also post your question on the [Kubernetes Slack #external-dns](https://kubernetes.slack.com/archives/C771MKDKQ).
+You can also post your question on the [Kubernetes Slack #external-dns](https://slack.k8s.io/?channel=external-dns).
```

Comment:
I don't think this is the right URL, I am logged into slack and I only get asked for my email:

![Screenshot 2023-10-27 at 13 41 41](https://github.com/kubernetes-sigs/external-dns/assets/683988/9f8644f6-4ad9-4cec-9c03-47dad6b66905)

Reply:
I think you should be authenticated before being able to access the channel

Reply:
The new URL doesn't work for me and the old one does.

Reply:
I think I got it. It's not k8s.slack.com, it's kubernetes.slack.com
I also managed to found their doc: https://api.slack.com/reference/deep-linking#app_channel
=> Complete url : https://kubernetes.slack.com/app_redirect?channel=external-dns
On my side, in a private window, at first it asked for login and second times it opened me the

Reply:
https://kubernetes.slack.com/channels/external-dns seems to work, also.